build.gradle

```diff
@@ -62,10 +62,10 @@ dependencies {
     // Solace Messaging API for Java Dependencies
     implementation group: 'com.solacesystems', name: 'sol-jcsmp', version: '10.+'
     // new improved logging framework, log4j2
-    implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.+'
```

Comment:
I think we should just leave `2.+` for these 3 dependencies. log4j is mature enough that they shouldn't be making backwards breaking changes as part of minor versions.